core/repository/api/src/main/java/org/eclipse/rdf4j/repository/config/AbstractRepositoryImplConfig.java

```diff
@@ -66,6 +67,8 @@ public Resource export(Model model) {
 
 		if (type != null) {
 			model.add(implNode, CONFIG.Rep.type, literal(type));
+			model.add(implNode, REPOSITORYTYPE, literal(type));
```

Comment:
I'm really not a fan of this: we're now creating config files that duplicate data, and we re-introduce the deprecated vocabulary into new files. The setup we went for was to make sure that any new files only use the new vocabulary, and provide helpers and utilities to make reading of both old and new files doable. 

I'd like to understand a little better why just tweaking GraphDB to work with the new vocabulary (or at least support both) on _its_ side isn't an option here.

Reply:
The issue isn't really GraphDB, but that the introduction of the new vocabulary amounts to a breaking change for anyone who is using the current (legacy) vocabulary.